sdk/kusto/setup.ps1

```diff
@@ -0,0 +1,30 @@
+Import-Module Az.Kusto
```

Comment:
I believe this is only for the tests in `Azure.ResourceManager.Kusto`. If that's correct, could you please do me a favor to make the project and file structure clearer? I think what we can do here include:

- Create a new folder `Prerequisite` under `sdk/kusto/Azure.ResourceManager.Kusto/tests`.
- Move the `setup.ps1`, `test-resources.bicep` and the folder `data` to `Prerequisite`
- Create a new markdown file under `Prerequisite` to tell the coworkers what setup.ps1 is used for, what they need to install and update when they want to run the script. Here is one [example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/prerequisites/prerequisite%20readme.md) from another RP.